EvidenceApi/V1/Domain/Claim.cs

```diff
@@ -19,5 +19,6 @@ public class Document
         public Guid Id { get; set; }
         public int FileSize { get; set; }
         public string FileType { get; set; }
+        public string DocumentDescription { get; set; }
```

Comment:
This is in the `Document` response so it feels like it should just be `Description`, similar to how the name is just `Name`. I believe it's `DocumentDescription` in the request to `documents-api` and `Document.Description` in the response.